pact/Pact/Core/Repl/Runtime/ReplBuiltin.hs

```diff
@@ -373,15 +375,17 @@ envExecConfig :: ReplCEKEval step => NativeFunction step ReplCoreBuiltin SpanInf
 envExecConfig info b cont handler _env = \case
   [VList s] -> do
     s' <- traverse go (V.toList s)
-    replEvalEnv . eeFlags .= S.fromList s'
-    let reps = PString . flagRep <$> s'
+    let (knownFlags, _unkownFlags) = partitionEithers s'
+    -- TODO: Emit warnings of unkown flags
```

Comment:
@jmcardon I think we should address this in a separate PR

Reply:
I agree here. This is actually kind of an odd piece of code. I _think_ we're just accumulating flags that make sense, but that begs the question of why this isn't just accumulating instead of lifting what we don't want into rights and what we want into lefts. I think we should error out if there's unknown flags tbqh

Reply:
Im not sure about this one: silently ignoring only specific flags doesn't seem right to me. I'm in favour of ignoring them all for now and adding warnings to it. @emilypi wdyt?